paths/electron/starting-with-electron/02-electron-boilerplate.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Electron Boilerplate
+permalink: /electron/starting-with-electron/boilerplate
+next-page: /electron/starting-with-electron/init-git-repository
+facilitator: false
+sidebar:
+  nav: "electron-getting-starting"
+main-content: |
+
+  Right now, our directory has no files. We want it to be set up as a base electron boilerplate called [electron-forge](https://electronforge.io/). To do that, we need to go through a few steps.
+
+  Important files to watch out for:
+  - package.json
+  - src/index.html
+  - src/index.json
+
```

Comment:
It appears that these files aren't being rendered as a list. If them being a list was the intention, there should be a blank line between `for:` and `- package.json`.

With blank line:

<img width="351" alt="screen shot 2017-08-07 at 1 26 03 pm" src="https://user-images.githubusercontent.com/1038121/29044375-fd9d53f2-7b73-11e7-8b9e-97a1288caa84.png">

Without blank line:

<img width="662" alt="screen shot 2017-08-07 at 1 26 43 pm" src="https://user-images.githubusercontent.com/1038121/29044398-15df0f82-7b74-11e7-87b3-38ce8d92e7c6.png">